tests/acceptance/features/cliDataExporter/export.feature

```diff
@@ -74,3 +74,12 @@ Feature: An administrator wants to export the files of his user using
     When user "unknown" is exported to path "/tmp/fooSomething" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "Could not extract user metadata for user"
+
+  @issue-209
+  Scenario: exporting with empty trash bin
```

Comment:
is this a bug demonstration scenario?

Reply:
In this scenario there is no bug occurs but in another scenario which is another feature file
"exportFileWithEmptyTrashBinDataExporterIssue209.feature" here is senario for bug demonstrartion scenario